src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -165,82 +150,89 @@ public void StreamCopyError(StreamCopyingThread who, String error) {
         }
     }
 
-    public int GetLocalHostPort() {
+    private  int GetLocalHostPort() {
```

Comment:
Per my previous suggestion I would actually get rid of mHTTPPort all together and just use GetLocalHostPort directly for everything and have it call the srvSocket and get the port directly.